app/controllers/application_controller.rb

```diff
@@ -85,4 +85,13 @@ def after_sign_in_path_for(user)
       referer || default_path
     end
   end
+
+  # Render a turbo-stream response without all the boilerplate
+  def render_turbo_response(*args)
```

Comment:
We don't need to support non-JS client in the admin, but we should be careful about depending on it on public pages.